src/Input.cpp

```diff
@@ -153,24 +142,61 @@ struct Gosu::Input::Impl
                 }
                 break;
             }
+            case SDL_JOYAXISMOTION: {
+                if (SDL_IsGameController(e->jaxis.which)) {
+                    break; // Ignore as triggers are reported -1..1 instead of 0..1
+                } else {
```

Comment:
Looks like rake doesn't like this line and wants the `else` to be on its own line after the `}`, which is causing the CI to fail. Not sure why rake is so stingy about formatting, but I guess it just is?

Reply:
I keep forgetting to run the formatter... corrected now. :slightly_smiling_face: